spec/factories/profiles.rb

```diff
@@ -0,0 +1,23 @@
+FactoryBot.define do
+  factory :profile do
+    user
+    attendance { "attendance" }
+    blog { "blog" }
+    facebook { "facebook" }
+    feminism { "feminism" }
+    gravatar_email { "gravatar_email" }
+    linkedin { "linkedin" }
+    projects { "projects" }
+    pronouns { "pronouns" }
+    reasons { "reasons" }
+    show_name_on_site { false }
+    show_projects { false }
+    show_reasons { false }
+    show_skills { false }
+    skills { "skills" }
+    summary { "summary" }
+    twitter { "twitter" }
+    website { "website" }
+    user_id { "user_id" }
+  end
+end
```

Comment:
I feel like the linter should catch this... when I run `bundle exec standardrb` locally I get 10+ files changed. Maybe I will do that in a later PR...